py_zipkin/thread_local.py

```diff
@@ -42,10 +44,7 @@ def get_zipkin_attrs():
     :rtype: :class:`zipkin.ZipkinAttrs`
     """
     from py_zipkin.storage import ThreadLocalStack
-    warnings.warn(
-        'Use py_zipkin.stack.ThreadLocalStack().get',
-        DeprecationWarning,
-    )
+    log.warning('Use py_zipkin.stack.ThreadLocalStack().get')
```

Comment:
Since this is no longer a `DeprecationWarning`, can you explicitly call out the deprecation? Same for the other ones.

Reply:
good point